pages/linux/numactl.md

```diff
@@ -0,0 +1,16 @@
+# numactl
+
+> Control NUMA policy for processes or shared memory.
+> More information: <https://man7.org/linux/man-pages/man8/numactl.8.html>.
+
+- Run a command on node 0 with memory allocated on node 0 and 1:
+
+`numactl --cpunodebind={{0}} --membind={{0,1}} -- {{command}} {{command_arguments}}`
+
+- Run a command on CPUs (cores) 0-4 and 8-12 of the current cpu-set:
```

Comment:
The man page says `cpuset` everywhere...
```suggestion
- Run a command on CPUs (cores) 0-4 and 8-12 of the current cpuset:
```

Reply:
Yup, cpuset is correct: https://man7.org/linux/man-pages/man7/cpuset.7.html